src/romi-robot.ts

```diff
@@ -645,6 +659,22 @@ export default class WPILibWSRomiRobot extends WPILibWSRobotBase {
                 encoderInfo.reportedValue += delta;
                 encoderInfo.lastRobotValue = encoderValue;
 
+                const currTimestamp = Date.now();
+
+                // Calculate the period
+                if (encoderInfo.lastReportedTime !== undefined) {
+                    const timespanMs = currTimestamp - encoderInfo.lastReportedTime;
+                    // Period = (approx) timespan / delta
+                    if (delta === 0) {
+                        encoderInfo.reportedPeriod = Number.MAX_VALUE;
```

Comment:
Should this be 0 instead of max value?

Reply:
The rate would be 0, but the period would be max value (since we're essentially waiting an infinite amount of time for another encoder edge). 

(it's also how the HAL encoder is initialized: https://github.com/wpilibsuite/allwpilib/blob/b60eb1544b3a04443060db51f2f3f299cb1eca4b/hal/src/main/native/sim/Encoder.cpp#L178)